doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                            <br/>
+                            Did you get all the reasons above and think of others?
+                        </div>
+                    </div>
+                </div>
+                <br/>
+                Scenario 2
+                <br/>Daisy has been developing some code, she now wants help from Mike to develop the code further using Mikes expertise.
+                <br/>
+                Mike has taken a copy of the code and Daisy is still developing it.
+                <br/>
+                Mike tries to merge his code in to the trunk version that Daisy has committed but the code has been updated and there is a conflict.
+                <br/>
+                Using Fcm they can see the conflict and resolve it using a built in tool.
+                <br/>
+                <br/>
+                Scenario 3
+                <br/>
+                Tim and Alan need to re-run a older version of some code as a customer, DarkStar comics, wants to reproduce data from three years ago as they have lost the data and need to run the code with the setting from three years ago for the Inland Revenue.
+                <br/>
+                Tim is using a version control system and can extract the code from the correct time period and reproduce the data.
+                <br/>
+                Alan did not use version control and just changes the code every time the rates changed.
+                <br/>
+                He has to look-up the correct values and rewrite the code before he can reproduce the data.
+                <br/>
+                <br/>
+                Scenario 4
+                <br/>
+                Marsha and Twist work in fashion marketing, Marsha has provided Twist with some market changes forecasting code from a version control repository.
+                <br/>
+                The code forecasts the supply and demand trend for the coming month based on previous years data and trends.
+                <br/>
+                Twist made some changes to her code in her local/working copy and ran the code.
+                <br/>
+                This produced some very unrealistic results and Twist realised that she had made some fundamental mistakes.
+                <br/>
+                As she had not committed her changes to the version control repository then she can just revert her local changes and start again.
+                <br/>
+                <br/>
+                Scenario 5
+                <br/>
+                Following on from scenario 4 Twist this time made the changes and the code failed but this time she had committed the code to the repository.
+                <br/>
+                It is all okay as she can go back to the last checked in version that she know worked as expected using version control.
```

Comment:
It is all okay as she can use version control to go back to the last checked in version that worked as expected.